chef/cookbooks/barclamp/libraries/nic.rb

```diff
@@ -874,6 +874,7 @@ def parent
 
     def destroy
       super
+      ::Kernel.system("ifdown #{@nic}")
```

Comment:
Hm, wouldn't this belong into `kill_nic` in the network cookbook. I think the goal once was to keep nic.rb free of stuff related to the distro's network management framework (even though `ifdown` might be somewhat portable).

And even though the bugreport is about vlans, I guess the call makes sense for other types interfaces as well. But I might be overlooking something here.